crates/xtask/src/commands/prep/installers.rs

```diff
@@ -0,0 +1,75 @@
+use anyhow::{anyhow, Context, Result};
+use camino::{Utf8Path, Utf8PathBuf};
+use regex::bytes::Regex;
+
+use std::{fs, str};
+
+use crate::utils::{self, PKG_VERSION};
+
+/// installers::update_versions prepares our curl/iwr installers
+/// with the Cargo.toml version
+pub(crate) fn update_versions() -> Result<()> {
+    utils::info("updating shell installer versions.");
+    let scripts_dir = get_binstall_scripts_root()?;
+    update_nix_installer_version(&scripts_dir)?;
+    update_win_installer_version(&scripts_dir)
+}
+
+// update_nix_installer_version updates our curl installer with the Cargo.toml version
```

Comment:
you can technically do `///` for this line here as well, since local docs will still print function descriptions, even thought the function is private. You can also do `//` on `pub(crate)` functions, because that will never be accessible to the public. Just a small tidbit of info!

![image](https://user-images.githubusercontent.com/8107784/121524030-29c7d800-c9f7-11eb-8254-24648ff2352f.png)